### PR TITLE
feat: added avatar and button component in documentation

### PR DESCRIPTION
**Pull request by ParthParolekar**
added avatar and button component in documentation
live: https://deploy-preview-8--electtron.netlify.app/documentation

### CODE REVIEW

File: documentation.html [hunk 1]
```diff
@@ -155,6 +156,328 @@ <h3 class="component-subtitle">Outlined Alert</h3>
...
+          />
+        </div>
+
+        <div class="component-code">
+          <iframe
+            src="https://carbon.now.sh/embed?bg=rgba%280%2C0%2C0%2C1%29&t=seti&wt=none&l=auto&width=766&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cimg%250A%2520%2520%2520%2520src%253D%2522%255Bimage%2520source%255D%2522%250A%2520%2520%2520%2520alt%253D%2522Avatar%2522%250A%2520%2520%2520%2520class%253D%2522avatar%2522%250A%252F%253E"
+            style="
+              width: 766px;
+              height: 167px;
+              border: 0;
```
Comment (vrx29) [inline documentation.html:186]: try not to use inline styling,

Comment (ParthParolekar) [reply documentation.html:186]: Those are styles that come with the carbon code snippet when copying the `iframe`

File: Documentation/Docs-CSS/main-section.css [hunk 2]
```diff
@@ -5,7 +5,7 @@
 .component-section {
   width: 90vw;
   padding: 1rem;
-  margin: 1rem auto;
+  margin: 3rem auto;
   height: fit-content;
   background-color: var(--secondary);
   color: var(--primary);
```
Comment (venkateshwarans) [inline Documentation/Docs-CSS/main-section.css]: Nice play with the dark background. One suggestion - dark blue to the font color is a bit too jarring. 
You can also see the contrast ratio in the dev tools giving us a warning. See if you can play with some other colors for the fonts. 

image

**Review by venkateshwarans — APPROVED** [review]
Added a suggestion. Rest everything looks good. Great work

File: Documentation/Docs-CSS/main-section.css (hunk 2 above)
Comment (ParthParolekar) [reply Documentation/Docs-CSS/main-section.css]: Oh, right! I wanted to make all the text on dark background appear white but I think I forgot to remove the color property from the `component-section`. I will change it right away. Thanks for the review!

**Review by Dev-AkshatGupta — APPROVED** [review]
LGTM great work Parth

**Comment by ParthParolekar** [comment]
Thanks for the review @vrx29 @venkateshwarans @Dev-AkshatGupta 
Merging now

**Pull request merged**